server/tests/users.test.js

```diff
@@ -45,11 +46,166 @@ describe("POST /api/v1/users/register", () => {
         done();
       });
   });
+}); 
+/*
+ * @route   POST api/v1/users/register
+ * @params  {name, email}
+ * @desc    Register user Missing 1 Param should send 400
+ * @access  Public
+ */
+describe("POST /api/v1/users/register", () => {
+  const body = {
+    name: "Johnny Doe",
+    email: "JohnDoeMissingParm@Gmail.com"
+
+  };
+  it("Should Respond With 400 Created", done => {
+    request
+      .post("/api/v1/users/register")
+      .send(body)
+      .set("Accept", "application/json")
+      .expect(400)
+      .end(err => {
+        if (err) return done(err);
+        done();
+      });
+  });
 });
+/*
+ * @route   POST api/v1/users/register
+ * @params  {name, email, password}
+ * @desc    Register user using same email should send 400
+ * @access  Public
+ */
+describe("POST /api/v1/users/register", () => {
+  const body = {
+    name: "Johnny Doe",
+    email: "JohnDoe@Gmail.com",
+    password: "123456"
 
-// Test -> Register Missing 1 Param SHOULD SEND 400
-// Test -> Register with existing email SHOULD SEND 400
-// Test -> User Login: Success
-// Test -> User Login: Fail Incorrect Password SEND 400
+  };
+  it("Should Respond With 400 Created", done => {
+    request
+      .post("/api/v1/users/register")
+      .send(body)
+      .set("Accept", "application/json")
+      .expect(400)
+      .end(err => {
+        if (err) return done(err);
+        done();
+      });
+  });
+});
+/*
+ * @route   POST api/v1/users/login
+ * @params  {email, password}
+ * @desc    Tests successful login, should send 200
+ * @access  Public
+ */
+describe("POST /api/v1/users/login", () => {
+  const body = {
+    email: "JohnDoe@Gmail.com",
+    password: "123456"
+  };
+  it("Should Respond With Success", done => {
+    request
+      .post("/api/v1/users/login")
+      .send(body)
+      .set("Accept", "application/json")
+      .expect(200)
+      .end((err,res) => {
+        if (err) return done(err);
+        else
+        {
+          token =res.body.token;
```

Comment:
Small consistency change. I have an extension on VSCode named [Prettier](https://prettier.io/) which I highly recommend. I will share you guys my VSCode settings and extensions in our next meeting to avoid small consistency changes.

Change
- token =res.body.token;
- token **= res.body.token;**